app/controllers/application_controller.rb

```diff
@@ -1,55 +1,99 @@
 class ApplicationController < ActionController::Base
+
```

Comment:
Looks good to me.
Build failed.
170 files inspected, 17 offenses detected
The command "bundle exec rubocop" failed and exited with 1 during .